packages/cli/test/functional/test_site/expected/testPanels.html

```diff
@@ -0,0 +1,288 @@
+                      </ul>
+                    </li>
+                  </ul>
+                </li>
+                <li>
+                  <div class="site-nav-default-list-item site-nav-list-item-0" onclick="handleSiteNavClick(this)">Test line break in navigation layout
+
+                    <i class="site-nav-dropdown-btn-icon" onclick="handleSiteNavClick(this.parentNode, false); event.stopPropagation();">
+                      <span class="glyphicon glyphicon-menu-down" aria-hidden="true"></span>
+                    </i>
+                  </div>
+                  <ul class="site-nav-dropdown-container site-nav-list">
+                    <li class="site-nav-custom-list-item site-nav-list-item-1">Nested line break text ✂️</li>
+                    <li>
+                      <div class="site-nav-default-list-item site-nav-list-item-1" onclick="handleSiteNavClick(this)"><a href="/test_site/index.html">Nested line break href</a>
+
+                        <i class="site-nav-dropdown-btn-icon" onclick="handleSiteNavClick(this.parentNode, false); event.stopPropagation();">
+                          <span class="glyphicon glyphicon-menu-down" aria-hidden="true"></span>
+                        </i>
+                      </div>
+                      <ul class="site-nav-dropdown-container site-nav-list">
+                        <li class="site-nav-custom-list-item site-nav-list-item-2">Nested Nested line break text ✂️</li>
+                      </ul>
+                    </li>
+                    <li>
+                      <div class="site-nav-default-list-item site-nav-list-item-1" onclick="handleSiteNavClick(this)">Nested line break dropdown menu
+
+                        <i class="site-nav-dropdown-btn-icon" onclick="handleSiteNavClick(this.parentNode, false); event.stopPropagation();">
+                          <span class="glyphicon glyphicon-menu-down" aria-hidden="true"></span>
+                        </i>
+                      </div>
+                      <ul class="site-nav-dropdown-container site-nav-list">
+                        <li class="site-nav-custom-list-item site-nav-list-item-2">Line break item 2 📘</li>
+                      </ul>
+                    </li>
+                  </ul>
+                </li>
+              </overlay-source>
+            </site-nav>
+          </div>
+        </div>
+      </overlay-source>
+      <div id="content-wrapper">
+        <p><strong>Panel with attributes</strong></p>
+        <panel><template #header>
+            <p>Correct header</p>
+          </template>
+          Lorem ipsum...
+        </panel>
+        <br>
+        <panel><template #header>
+            <p>Correct header</p>
+          </template><template #_alt>
+            <p>Alternative header</p>
```

Comment:
Thanks for removing the `alt` attribute from the tests! Remember to update these files as well :)

Reply:
Thank you for taking note of this! I will remember to update tests next time :sweat_smile: 

Fixed in [eef8681](https://github.com/MarkBind/markbind/pull/2053/commits/eef8681472a7f46b39bb401ae7138e0172e8787b) :+1: